### DISCUSSION TITLE
State Timeline: offer the possibility to remove the name of the series

### DISCUSSION THREAD

**Discussion by ghost** [grafana/grafana#63864] [Dashboards, Panels & Visualizations]
We have the usecase where we have only one data serie displayed on a state timeline visualization. Would be great if we could remove the name of the data serie (see circled in red below) as it only takes space that could be used to extend the timeline. 

![image](https://user-images.githubusercontent.com/29061201/221876012-246fe24b-db81-4ecf-b97b-5a02062e68e9.png)

**Comment by ghost** [comment]
Did not see the option, maybe there's a way to do it and I missed it? If not, that could be an interesting feature.

**Reply by lukasztyrala** [reply]
What type of data source are you using? (Some data sources allow alias for names in query parameters.)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82695. Feel free to continue the discussion around this there. Thank you!
